### ISSUE TITLE
Yellow Ribbon tool - focus does not move on search

### ISSUE THREAD

**Issue by andaleliz** [department-of-veterans-affairs/va.gov-team#7133]
## What happened?
When you conduct a search, the focus stays on the search button after clicking

## Specs:
- Device: Macbook
- Browser: Chrome
- Test User _(if applicable)_: n/a

## Steps to Reproduce
- URL: https://staging.va.gov/education/yellow-ribbon-participating-schools/
- Test User _(if applicable)_: n/a
1.  Visit the URL above
2. Conduct a search
3. Observe the focus border remaining on the button.

## Desired behavior
Focus should go to the h1 element when the search results load.

## Acceptance Criteria
- [ ] Focus goes to h1 element
- [ ] h1 does not have yellow outline (needs to get focus so screen readers can hear it)
- [ ] h1 has nested `<span class=“ads-u-visibility--screen-reader”> returned [number] schools</span>`

 with

## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [x] **Attached to Epic** (what body of work is this a part of? possibly `Ongoing Maintenance`)
- [x] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `triage`, `tools-improvements`)
- [x] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [x] **Labeled with `Bug`**

**Comment by andaleliz** [comment]
@kelsonic related, but not in an issue yet because I'm not sure if it's still "out there" to be worked on.  Want to bring it up here in case it would be easy to address while you're working on this.

Per the [dev notes in this screen](https://vsateams.invisionapp.com/share/P4W04QPWHTK#/405289244_Results_-__dev_Notes), the h1 should also have a `span` element inside:
`<span class=“ads-u-visibility--screen-reader”> search returned [number] schools</span> `

This is something @1Copenut and I collaborated on to improve screen reader UX, so when the search results are displayed, a user utilizing AT would hear "Yellow Ribbon school search results search returned 409 schools"

(that does seem a bit verbose to me now that I'm thinking about it again, but we can stick with that for now, unless @1Copenut has additional guidance)

**Comment by 1Copenut** [comment]
@andaleliz I think we could shorten the `<span class=“ads-u-visibility--screen-reader”>` text to just "returned [number] schools, so the total `<h1>` text would be read as "Yellow Ribbon search results returned [number] schools, heading level one". That is a verbose title, but it's also highly descriptive. Then you have the "Showing 1-10 of [Number]" further down the page. If users offer feedback it's too dense, we can shorten up the H1, but I think we're making the right call for the first go-round.

**Comment by andaleliz** [comment]
@1Copenut That sounds great.  @kelsonic let us know if you have any questions

**Comment by kelsonic** [comment]
@andaleliz Awesome! Would you mind updating the acceptance criteria if need be?

**Comment by andaleliz** [comment]
@kelsonic adding that span to the h1 isn't part of the "bug" that's called out in this issue.  I can def update the AC but not sure if it matters to separate out the different categories of work or not.

**Comment by andaleliz** [comment]
@kelsonic went ahead and added it here but lmk if you want me to break it out into an other issue

**Comment by kelsonic** [comment]
@andaleliz Thank you! 🙂

**Comment by MickinSahni** [comment]
@kelsonic could you throw an estimate on this when you have a moment? i'd like it to be available for pickup this sprint if possible. 🤞
